docs/manual/node-steps/builtin.md

```diff
@@ -135,6 +135,51 @@ Data
 Format
 : Format for the data. One of: `properties`, `json`, or `yaml`
 
+### HTTP Node Step
+
+Performs an HTTP Request with or without authentication (per node)
+
+![HTTP Node Step](~@assets/img/http-request-step.png)
+
+#### Configuration
+
+Remote URL
+: The URL to which to make the request.
+
+HTTP Method
+: The method that you want to be performed on the URL
+
+Headers
+: Any headers you wish to include
+
+Body
+: Any body you want
+
+Request Timeout
+: How long to wait for the request to complete before failing
+
+![HTTP Node Step - Authentication](~@assets/img/http-node-step-auth.png)
+
+#### Authentication 
+
+Authentication
+: Method of authentication to use
```

Comment:
Are there different options exist for the various Authentication methods?  Let's list the options they have here so it's covered and indexed by the doc search.